features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/ViewUtilsInternalTest.kt

```diff
@@ -182,5 +225,149 @@ internal class ViewUtilsInternalTest {
+            messageBuilder = msgCaptor.capture(),
+            throwable = anyOrNull(),
+            onlyOnce = anyOrNull()
+        )
+
+        assertThat(levelCaptor.firstValue).isEqualTo(InternalLogger.Level.ERROR)
+        assertThat(targetCaptor.firstValue).isEqualTo(InternalLogger.Target.MAINTAINER)
+        assertThat(msgCaptor.firstValue()).isEqualTo(expectedError)
+    }
+
+    @Test
+    fun `M return bounds W resolveCompoundDrawableBounds() { for left drawable }`() {
+        // When
+        val actualBounds = testViewUtilsInternal.resolveCompoundDrawableBounds(
+            mockTextView,
+            mockDrawable,
+            0f,
+            0,
+            mockInternalLogger
+        )
+
+        // Then
+        assertThat(actualBounds?.x).isEqualTo(mockTextView.paddingStart.toLong())
+        assertThat(actualBounds?.y).isEqualTo(
+            (mockTextView.height / 2 - mockDrawable.intrinsicHeight / 2).toLong()
+        )
+    }
+
+    @Test
+    fun `M return bounds W resolveCompoundDrawableBounds() { for top drawable }`() {
+        // When
+        val actualBounds = testViewUtilsInternal.resolveCompoundDrawableBounds(
+            mockTextView,
+            mockDrawable,
+            0f,
+            1,
+            mockInternalLogger
+        )
+
+        // Then
+        assertThat(actualBounds?.x).isEqualTo(
+            (mockTextView.width / 2 - mockDrawable.intrinsicWidth / 2).toLong()
+        )
+        assertThat(actualBounds?.y).isEqualTo(
+            mockTextView.paddingTop.toLong()
+        )
+    }
+
+    @Test
+    fun `M return bounds W resolveCompoundDrawableBounds() { for right drawable }`() {
+        // When
+        val actualBounds = testViewUtilsInternal.resolveCompoundDrawableBounds(
+            mockTextView,
+            mockDrawable,
+            0f,
+            2,
+            mockInternalLogger
+        )
+
+        // Then
+        assertThat(actualBounds?.x).isEqualTo(
+            (mockTextView.width - (mockDrawable.intrinsicWidth + mockTextView.paddingEnd)).toLong()
+        )
+        assertThat(actualBounds?.y).isEqualTo(
+            (mockTextView.height / 2 - mockDrawable.intrinsicHeight / 2).toLong()
+        )
+    }
+
+    @Test
+    fun `M return bounds W resolveCompoundDrawableBounds() { for bottom drawable }`() {
+        // When
+        val actualBounds = testViewUtilsInternal.resolveCompoundDrawableBounds(
+            mockTextView,
+            mockDrawable,
+            0f,
+            3,
+            mockInternalLogger
+        )
+
+        // Then
+        assertThat(actualBounds?.x).isEqualTo(
+            (mockTextView.width / 2 - mockDrawable.intrinsicWidth / 2).toLong()
+        )
+        assertThat(actualBounds?.y).isEqualTo(
+            (
+                mockTextView.height -
+                    (
+                        mockDrawable.intrinsicHeight +
+                            mockTextView.paddingBottom
+                        )
+                ).toLong()
```

Comment:
seems like bad formatting

Reply:
yes, ktlint keeps "fixing" this line to look like this. I'll find something that it won't modify